src/login/logind.c

```diff
@@ -66,11 +56,29 @@ static Manager *manager_new(void) {
 
```

Comment:
hmm, m->power_key_ignore_inhibited must be rest to false explicitl here now, too. Same for suspend_key_ignore_inhibted, and hibernate_key_ignore_inhibited.